libcloudforensics/gcp.py

```diff
@@ -534,6 +504,35 @@ def __ListByLabel(self, labels_filter, service_object, filter_union):
           previous_request=request, previous_response=response)
     return resource_dict
 
+  @staticmethod
+  def _ReadStartupScript():
+    """Read and return the startup script that is to be run on the forensics VM.
+
+    Users can either write their own script to install custom packages,
+    or use the provided one. To use your own script, export a STARTUP_SCRIPT
+    environment variable with the absolute path to it:
+    "user@terminal:~$ export STARTUP_SCRIPT='absolute/path/script.sh'"
+
+    Returns:
+      str: the script to run
+
+    Raises:
+      OSError: If the script cannot be opened, read or closed.
+    """
+    try:
+      startup_script = os.environ.get('STARTUP_SCRIPT')
+      if not startup_script:
+        # Use the provided script
+        startup_script = os.path.dirname(os.path.realpath(__file__)) + \
+                         '/../scripts/startup.sh'
```

Comment:
Use `os.path.dirname` to go up a level rather than `..`: https://docs.python.org/3/library/os.path.html#os.path.dirname (you can use it as many times as you want to go up as many levels as you want)